__tests__/app.test.js

```diff
@@ -16,6 +19,99 @@ afterAll(() => {
+                    })
+                })
+            })
+            test("each object in the 'articles' array has the expected keys", () => {
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                    const { articles } = body
+                    expect(articles.length).toBeGreaterThan(0)
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author")
+                        expect(article).toHaveProperty("title")
+                        expect(article).toHaveProperty("article_id")
+                        expect(article).toHaveProperty("topic")
+                        expect(article).toHaveProperty("created_at")
+                        expect(article).toHaveProperty("votes")
+                        expect(article).toHaveProperty("article_img_url")
+                        expect(article).toHaveProperty("comment_count")
+                    })
+                })
+            })  
+            test("the value of 'comment_count' is equal the number of comments for a given article", () => {
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                    const { articles } = body
+                    expect(articles.length).toBeGreaterThan(0)
+                    articles.forEach((article) => {
+                        const expectedCount = comments.filter((comment) => {
+                            return comment.article_id == article.article_id
+                        }).length
+                        expect(article.comment_count).toBe(expectedCount)
+                    })
+                })
+            })  
+            test("the values of 'title', 'topic', 'author', 'body', 'created_at' and 'article_img_url' are retreived from the 'nc_snack_database'", () => {
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                    const { articles } = body
+                    expect(articles.length).toBeGreaterThan(0)
+
+                    valueStrings = expectedArticles.map((article) => {
+                        const { title, topic, author, body, created_at, article_img_url } = article
+                        return `${title},${topic},${author},${body},${Date(created_at)},${article_img_url}`
+                    })
+
+                    articles.forEach((article) => {
+                        const { title, topic, author, body, created_at, article_img_url } = article
+                        const valueString = `${title},${topic},${author},${body},${Date(created_at)},${article_img_url}`
+                        expect(valueStrings.includes(valueString)).toBe(true)
+                    })
+
+                })
+            }) 
```

Comment:
This does not need to be a test, the above tests would already check the data is what you expect.